src/utils/Options.ts

```diff
@@ -1,10 +1,10 @@
 module n3Charts.Utils {
   'use strict';
 
-  export interface IAxes {
-    x: AxisOptions;
-    y: AxisOptions;
-    y2?: AxisOptions;
+  export interface IAxesSet {
```

Comment:
I thought very long about this; for me it was confusing that it is not just axes (like multiple axis) but it has a special shape. It's an object in fact, that deserves a name